src/client/components/Google_login_button.components/GoogleLoginButton.css

```diff
@@ -0,0 +1,22 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&display=swap');
+
+.google_login_button {
+  display: flex;
+  align-items: center;
+  height: 3em;
+  font-family: 'Roboto', sans-serif;
+  font-family: 'Roboto Mono', monospace;
```

Comment:
You don't need to use font-family twise. you could do like this: font-family: 'Roboto Mono', 'Roboto', monospace

Reply:
Yes, I am thinking to add a control now for these different fonts. Good catch, thanks :)

Reply:
:) I don't think that you need to add a prop/control for that, the button will have just one font-family, that's what I understood from issue explanation. what I wanted to say is that you use: font-family: 'Roboto Mono', 'Roboto', monospace. all in one line instead of having 2 lines. like that if the browser fails to load the first font-family, it loads the second, and if it fails it loads the third.
Let me know what you think

Reply:
Yes, indeed. I do not know why I added these 2 same styles twice :/ I copied different times, maybe that's how. 
But yes, I understand your point. What I thought later regarding the control over the fonts is, I have seen some of our friends' code that way of giving different font-families to choose from. But if you think, it's unnecessary, I can change it to css style.

Reply:
Indeed, in some buttons it is necessary to have a control for the color, but in your case the color us static, so having a control for it is not needed :)